editor/project_manager/project_list.h

```diff
@@ -68,6 +69,17 @@ class ProjectListItemControl : public HBoxContainer {
 	bool is_hovering = false;
 	bool is_favorite = false;
 
+	enum VersionMatchType {
+		VER_PROJECT_IS_THIS_VER,
```

Comment:
```suggestion
		VERSION_MATCH_TYPE_PROJECT_IS_THIS_VER,
```
Technically the constants should be prefixed with full enum name.
Although it's super verbose here, so as long as it's _some_ prefix then it's probably fine as is.